migrations/0043_normalize_claim_creds.up.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE public.claim_creds ADD COLUMN created_at timestamp without time zone;
+ALTER TABLE public.claim_creds ADD COLUMN updated_at timestamp without time zone;
+
+create view normalized_claim_creds as (
```

Comment:
We may not want to call this one normalized.  I called the other one normalized because I was converting each element of the json array to it's own row

Reply:
what's a better name?

Reply:
seems this is just adding the `json_array_length(signed_creds_length)`, otherwise this view is the same as the original table, right?

Reply:
correct, including `COALESCE`

Reply:
Maybe claim_creds_redshift_staging? or claim_creds_with_default_timestamps?

Reply:
I like `claim_creds_redshift_staging`